packages/headless/src/features/case-assist/case-assist-analytics-actions.ts

```diff
@@ -10,14 +11,18 @@ import {
 } from './case-assist-analytics-selectors';
 
 export const logCaseStart = (): CaseAssistAction =>
-  makeCaseAssistAnalyticsAction(
-    'analytics/caseAssist/case/start',
-    (client, state) =>
-      client.logEnterInterface({
+  makeCaseAssistAnalyticsAction({
+    prefix: 'analytics/caseAssist/case/start',
+    __legacy__getBuilder: (client, state) => {
+      return client.logEnterInterface({
         ticket: caseAssistCaseSelector(state),
-      })
-  );
+      });
+    },
+    analyticsType: 'CaseAssist.Start',
```

Comment:
Apparently this should be `camelCase` instead of `PascalCase`.
The casing doesn't matter for the event protocol, but at least when sending it we should use the right casing. :)

Reply:
BTW it's okay that the TypeScript Type starts with a capital letter.
But not the `type` in the event.

Reply:
It is kinda weird to write the type in two multiple ways, one `CaseAssist.Start` and `caseAssist.start`.
I would prefer consistency in the naming to avoid any confusions.

Reply:
updated the names to be `camelCase`.